### PR TITLE
Added Marktplaats

### PR DESCRIPTION
**Pull request by DarkVillager**
premid1 1
premid2 1
premid3 1
premid4 1
premid5 1
premid6 1

### CODE REVIEW

**Review by SlowLife1661 — CHANGES_REQUESTED** [review]
Fix codacy issues?

**Comment by DarkVillager** [comment]
> Fix codacy issues?

I fixed all of them other than the any issues. Looking into how I can replace them if they need to be replaced.

File: websites/M/Marktplaats/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  },
+  "url": ["www.marktplaats.nl", "help.marktplaats.nl"],
+  "regExp": "([a-z0-9-]+[.])*marktplaats[.]nl[/]",
+  "description": {
+    "nl": "Geniet van tweede hands producten kopen & verkopen op marktplaats!",
+    "en": "Enjoy buying & selling second hand products on marktplaats!"
+  },
+  "service": "Marktplaats",
+  "version": "1.0.0",
+  "logo": "https://i.imgur.com/qDJphwb.jpg",
```
Comment (Bas950) [inline websites/M/Marktplaats/dist/metadata.json:15]: logo must be 512x512

File: websites/M/Marktplaats/presence.ts [hunk 2]
```diff
@@ -0,0 +1,131 @@
+const presence = new Presence({
+  clientId: "811572600294735902"
+}), browsingStamp = Math.floor(Date.now() / 1000);
+let search: any,
+title: any;
```
Comment (Bas950) [inline websites/M/Marktplaats/presence.ts:5]: no any, use types

**Review by Bas950 — CHANGES_REQUESTED** [review]

File: websites/M/Marktplaats/presence.ts (hunk 2 above)
Comment (DarkVillager) [reply websites/M/Marktplaats/presence.ts]: Fixed.

File: websites/M/Marktplaats/dist/metadata.json (hunk 1 above)
Comment (DarkVillager) [reply websites/M/Marktplaats/dist/metadata.json:15]: Wrong link will fix asap.

**Review by ririxidev — APPROVED** [review]
Looks fine for me!

**Review by Bas950 — APPROVED** [review]

**Pull request merged**